lib/ukdebug/gdbstub.c

```diff
@@ -1106,3 +1113,78 @@ int gdb_dbg_trap(int errnr, struct __regs *regs)
 
 	return r;
 }
+
+static int gdb_setup_backing_console(void)
+{
+	/* Ignore breakpoints if the stub is not enabled on the command line. */
+	if (gdb_backing_console_id == GDB_BACKING_CONSOLE_NONE) {
+		uk_pr_info("Backing console not set\n");
+		return -ENXIO;
+	}
+
+	/* Ignore breakpoints if the backing console is not registered.
+	 * Maybe the backing console will be initialized later.
+	 */
+	if (gdb_backing_console_id >= uk_console_get_device_count()) {
+		uk_pr_info("Backing console not available\n");
```

Comment:
`uk_pr_warn` or `uk_pr_err` instead? Maybe add also a bit more context (`gdbstub:`), like
`"gdbstub: Console device %d is not available\n"`

Reply:
Doesn't it show in the debug output that the message is emitted from `gdbstub.c`? Why add the `gdbstub: `

Reply:
True. Then just the message with `"con%"__PRIu16` instead of `Console device %d`